addons/survey/views/survey_templates.xml

```diff
@@ -186,6 +186,8 @@
 
             <div class="row">
                 <div class="offset-3 col-6 text-center mt16">
+                    <button t-if="survey.users_can_go_back and page != survey.page_ids[0]" type="submit" class="btn btn-secondary"
```

Comment:
As explained on Discord, back buttons are in fact part of the "survey_navigation" template.
It was part of a recent survey pimping.

Here is the related commit: https://github.com/odoo/odoo/pull/46768/commits/0bc89dbb841424680684142377e1f596bf8d3c5a

(I also get lost in survey all the time, don't worry).